localstack/services/apigateway/integration.py

```diff
@@ -692,7 +691,7 @@ def invoke(self, invocation_context: ApiInvocationContext):
             queue_url = f"{config.internal_service_url()}/{account_id}/{queue}"
             new_request = f"{payload}&QueueUrl={queue_url}"
 
-        url = urljoin(config.internal_service_url(), f"{get_aws_account_id()}/{queue}")
+        url = urljoin(config.internal_service_url(), f"{account_id}/{queue}")
```

Comment:
Oh, damn, this was actually a bad bug 😬 shouldn't this URL depends on the SQS URL strategy actually?

Reply:
Not necessarily, URL strategy controls how queues will be formatted in outputs. SQS will still recognise other queue URL formats if presented. Moreover the use of `config.internal_service_url()` prevents us from using the standard and domain strategies.

But you're right in another way, the region is currently omitted in the queue URL. I've added it with 1b4fe6a04. Great catch, thanks!